src/workloads/execution/MultiPlanningReadsALotOfData.yml

```diff
@@ -0,0 +1,98 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
```

Comment:
I'm curious what the difference between server-execution and query-execution is in terms of Owner. For my perf workloads, I selected query-execution as the team.

Reply:
I will change it to query-execution.
Server-execution is more about StorEx team, I guess.